src/modules/roc_netio/target_uv/roc_netio/udp_receiver_port.cpp

```diff
@@ -263,5 +275,39 @@ void UDPReceiverPort::recv_cb_(uv_udp_t* handle,
     self.writer_.write(pp);
 }
 
+bool UDPReceiverPort::join_multicast_group_() {
+    if (!address_.miface()) {
+        return true;
+    }
+
+    if (int err = uv_udp_set_membership(&handle_, packet::ip_to_str(address_).c_str(),
```

Comment:
Do I understand correctly that uv_udp_set_membership() accepts both interface name and IP address? Is this behavior portable accross platforms?

Reply:
It accepts only interface address. I will convert interface name to interface address during port parsing.

Reply:
I've created #258. Take a look at the description of two new functions: `parse_endpoint_url()` and `resolve_address()`.

That task should be done separately from this PR, but the idea is that the resolving should be separated from parsing (at lease because resolving should stay in netio and depends on the network engine we use, and parsing is independent from that).

So for now, in this PR, I suggest to choose one of the two options:

* either don't perform interface name resolving at all and leave it for #258

* or create a simplified version of `netio::Transceiver::resolve_address()` that resolves interface name but doesn't resolve hostname; and we will finish it in #258